classes/patreon_wordpress.php

```diff
@@ -146,27 +237,40 @@ public static function getPatreonCreatorID() {
 						update_option('patreon-creators-access-token', $tokens['access_token']);
 					}
 
-					$user_response = $api_client->fetch_campaign_and_patrons();
+					$user_response = $api_client->fetch_creator_info();
 				}
 			}
 
 		}
+		
+		return $user_response;
+	}
+	
+	public static function getPatreonCreatorID() {
 
-        $creator_id = false;
+		$creator_info = self::getPatreonCreatorInfo();
 
-        if (array_key_exists('data', $user_response)) {
-            foreach ($user_response['included'] as $obj) {
-                if ($obj["type"] == "user") {
-                    $creator_id = $obj['id'];
-                    break;
-                }
-            }
-        }
+		if(isset($creator_info['included'][0]['id']))
+		{
```

Comment:
format ifs

Reply:
Committed:

https://github.com/codebard/patreon-wordpress/commit/a25c142224c1f28442c847e76185cb08420cb059

Reply:
looks like that commit doesn't address this